src/test/java/com/exactpro/epfast/template/assertion/TemplateAssert.java

```diff
@@ -0,0 +1,39 @@
+package com.exactpro.epfast.template.assertion;
+
+import com.exactpro.epfast.template.*;
+import org.assertj.core.api.AbstractAssert;
+
+import java.util.Objects;
+
+import static com.exactpro.epfast.template.assertion.InstrEqualityChecker.areSameInstructions;
+
+public class TemplateAssert extends AbstractAssert<TemplateAssert, Template> {
+
+    public TemplateAssert(Template actual) {
+        super(actual, TemplateAssert.class);
+    }
+
+    public static TemplateAssert assertThat(Template actual) {
+        return new TemplateAssert(actual);
+    }
+
+    public TemplateAssert isSameTemplateAs(Template template) {
```

Comment:
Rename to `isEqualToTemplate()`. "same" checks for reference equality in AssertJ. I was wrong with this name. Rename methods from `areSameXxx` to `areEqualXxx`